channel/src/main/java/com/synopsys/integration/alert/channel/email/EmailAddressHandler.java

```diff
@@ -93,7 +93,7 @@ public FieldUtility updateEmailAddresses(Long providerConfigId, MessageContentGr
         return new FieldUtility(fieldMap);
     }
 
-    public Set<String> getEmailAddressesForProject(ProviderProject project, boolean projectOwnerOnly) {
+    public Set<String> getEmailAddressesForProject(Long providerConfigID, ProviderProject project, boolean projectOwnerOnly) {
```

Comment:
`providerConfigID` and `providerConfigId` are both used in this class. We should stick to one capitalization strategy.